src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -0,0 +1,1125 @@
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Feature
+      </th>
+      <th>
+        New Relic agents
+      </th>
+      <th>
+        OpenTelemetry in New Relic
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        Collect metrics, traces, and logs
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Collect custom events
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        ❌ Not specified with OpenTelemetry
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Measure and track SLIs/SLOs
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡 Spans only at present
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Optimize application performance
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡 OpenTelemetry instrumentation varies significantly by language and is improving. New Relic agents have overall broader coverage of languages.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Discover root causes in complex systems
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡 Front-end and Kubernetes observability is limited
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Triage and fix errors and exceptions
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡 Front-end and Kubernetes observability is limited
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Understand system topology
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Monitor Kubernetes
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡 Better out-of-the-box experience with New Relic instrumentation
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Monitor serverless functions
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡 Better user experience with New Relic instrumentation, though setup is easier with OpenTelemetry for AWS Lambda
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Improve end-user experience
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        ❌ RUM (real user monitoring) specifications in progress
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Alert on any telemetry
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Detect and notify anomalies
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡 Automatic anomaly detection is presently limited to New Relic lookout
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Create dashboards
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        ✅
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Orchestrate and deploy observability tooling
+      </td>
+      <td>
+        ✅
+      </td>
+      <td>
+        🟡
```

Comment:
This is green for OTel at a high level. Specific minor feature differences belong in the detailed table, not here. There are advantages and disadvantages to both NR agent and OTel deployments--OTel has AWS ADOT, for example. Deployment markers are not a feature of tooling deployment, that's mixing up concerns here.